components/textfield/stories/template.js

```diff
@@ -196,9 +196,9 @@ export const Template = ({
 				})}
 			/>`
 		)}
-		${when(isLoading, () => ProgressCircle({
+		${when(isLoading, () => InfieldProgressCircle({
 			isIndeterminate: true,
-			size: "s",
+			size: size,
 			customClasses: customProgressCircleClasses,
```

Comment:
Should this be `customInfieldProgressCircleClasses`? Is that the intention here?